<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <attribute name="title">
      <string>主页</string>
     </attribute>
     <widget class="QTextBrowser" name="WelcomeText">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>671</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;欢迎来到啊哈工具箱！&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;请在上方的菜单栏中单击进入相应的功能。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DateCalculatorPage">
     <attribute name="title">
      <string>日期计算器</string>
     </attribute>
     <widget class="QDateEdit" name="Date1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>yyyy/M/d</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="Date2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>5</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>yyyy/M/d</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="IntervalDaysLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 相隔天数</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="IntervalDays">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> 天</string>
      </property>
      <property name="minimum">
       <number>-99999</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="Date1WeekDayDisplay">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>周六</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date2WeekDayDisplay">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>周三</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date1IsLeapYear">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>闰年</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date2IsLeapYear">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>闰年</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date1DayOfYearDisplay">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2000年的第1天</string>
      </property>
     </widget>
     <widget class="QLabel" name="Date2DayOfYearDisplay">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>220</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2000年的第5天</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="DateCalculatorHelp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>761</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;使用说明：本日期计算器中可编辑的部分有三个：第一日期、相隔天数（支持负数）、第二日期。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;当您编辑一个部分的时候，计算器将会计算并更改其他两个部分。&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;当您编辑两个日期中的任意一个，优先更改相隔天数；&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;当您编辑相隔天数，优先更改第二日期。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HTMLMDPreviewPage">
     <attribute name="title">
      <string>HTML/Markdown预览器</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>831</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPlainTextEdit" name="HTMLEdit">
         <property name="plainText">
          <string>左边是HTML编写区，右边是HTML预览区&lt;br&gt;

比如：&lt;br&gt;

&lt;h1&gt; 这是一个一级标题 &lt;/h1&gt; &lt;br&gt;
&lt;strong&gt; 这是一个粗体 &lt;/strong&gt; &lt;br&gt;
&lt;div style = &quot;color:red&quot;&gt; 红色的字体 &lt;/div&gt; &lt;br&gt;
&lt;div style = &quot;font-size:18px&quot;&gt; 18字号的字体 &lt;/div&gt; &lt;br&gt;
&lt;div style = &quot;font-family: Consolas&quot;&gt; Consolas!&lt;/div&gt; &lt;br&gt;
&lt;a href = &quot;https://www.cnblogs.com/crab-in-the-northeast&quot;&gt;超链接（顺便臭不要脸的推销一发自己的博客）&lt;/a&gt;

&lt;br&gt;

请注意本功能只支持纯文字预览。如果要求更高要求的预览，你可以试试在&lt;a href = &quot;about:blank&quot;&gt;about:blank&lt;/a&gt;（Chrome内核浏览器）里F12进行更高要求的HTML编写哦。</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="HTMLPreview">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;左边是HTML编写区，右边是HTML预览区&lt;br /&gt;比如：&lt;br /&gt;&lt;/p&gt;
&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;这是一个一级标题 &lt;/span&gt;&lt;/h1&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;这是一个粗体 &lt;/span&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;红色的字体 &lt;/span&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18px;&quot;&gt;18字号的字体 &lt;/span&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Consolas';&quot;&gt;Consolas!&lt;/span&gt;&lt;br /&gt;&lt;a href=&quot;https://www.cnblogs.com/crab-in-the-northeast&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;超链接（顺便臭不要脸的推销一发自己的博客）&lt;/span&gt;&lt;/a&gt; &lt;br /&gt;请注意本功能只支持纯文字预览。如果要求更高要求的预览，你可以试试在&lt;a href=&quot;about:blank&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;about:blank&lt;/span&gt;&lt;/a&gt;（Chrome内核浏览器）里F12进行更高要求的HTML编写哦。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="BtnHTMLPreview">
         <property name="text">
          <string>HTML</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="BtnMarkdownPreview">
         <property name="text">
          <string>Markdown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
